src/lib/jsonParseV2/parseContextAttribute.cpp

```diff
@@ -141,6 +141,22 @@ static std::string parseContextAttributeObject(const Value& start, ContextAttrib
     caP->valueType = orion::ValueTypeNone;
   }
 
+  // Is a date?
```

Comment:
"Is it a date"

Or just remove the comment. The code on the next line is pretty self-explanatory ...